ext/hiera.yaml

```diff
@@ -2,14 +2,8 @@
 :backends:
   - yaml
 :hierarchy:
-  - defaults
-  - "%{clientcert}"
-  - "%{environment}"
-  - global
+  - "nodes/%{clientcert}"
+  - common
 
 :yaml:
-# datadir is empty here, so hiera uses its defaults:
-# - /etc/puppetlabs/code/hieradata on *nix
-# - %CommonAppData%\PuppetLabs\code\hieradata on Windows
-# When specifying a datadir, make sure the directory exists.
-  :datadir:
+  :datadir: "/etc/puppetlabs/code/environments/%{environment}/hieradata"
```

Comment:
Following the same logic for this, it looks like the path should be `C:\ProgramData\PuppetLabs\code\environments\%{environment}\hieradata`

Based on (https://github.com/puppetlabs/puppet-specifications/blob/master/file_paths.md#puppet-agent-windows):

```md
C:\ProgramData\PuppetLabs\code *          # :codedir                     contents moved from C:\ProgramData\PuppetLabs\puppet\etc (:confdir)
    environments *                        # :environmentpath
```